src/_help/publish-documentation/create-and-manage-documentation.md

```diff
@@ -18,7 +18,7 @@ Creating documentation starts from your dashboard, using the button of the same
 ![](/images/help/doc-creation.png)
 
 | Name     | The name of the documentation as it will appear to those who have access to it.     |
-| Hub              | Here, you can specify whether you want to link this documentation to an existing hub or not. Hubs are collections of documentation.     |
+| Description              | Add a short description of your API documentation.     |
 | Slug | The slug allows you to define the URL of your documentation, in the format `https://bump.sh/your-username/your-slug`.     |
 | Documentation Access         | Documentation can be either public and accessible to everyone or private, visible only to you and invited members.     |
```

Comment:
We are missing several new fields here:
* the type of the API
* the prefered deployment method